documentation/IDTA-01003-a/modules/ROOT/pages/IDTA-01003-a_ChangeLog.adoc

```diff
@@ -17,15 +27,20 @@ Each subclause consists of three parts:
 * new elements in metamodel w.r..t previous version,
 * new, changed, or removed Constraints w.r.t previous version.
 
-== Changes V3.1 vs. V3.0.1
+
+
+== Changes V3.1 vs. V3.0.2
```

Comment:
Just as a reminder that the section "Changes V3.1 vs. V3.0.2 " should not be part of the v3.0.2 publication